Ullaakut/Bloggo
# App/split grid (#19)

## The goal of the PR

Splitting the initial css-grid into a flex-grid and a non-grid.

Part of #19

## How to test it

- open `index.html`
- check if a sidebar on the left is fixed
- check responsiveness of the page
- open in various browsers


## Screenshots

1. Mobile
![image](https://user-images.githubusercontent.com/38378975/49663557-680f6580-fa4f-11e8-9bcc-2269de4a4366.png)


2. Desktop
![image](https://user-images.githubusercontent.com/38378975/49663513-39918a80-fa4f-11e8-97d1-7c21917aba18.png)
